services/download/src/main/resources/application.yaml

```diff
@@ -19,6 +19,8 @@ services:
     retention-days: 14
     validation:
       allowed-report-types: ${ALLOWED_REPORT_TYPES:CONFIRMED_TEST,CONFIRMED_CLINICAL_DIAGNOSIS}
+    # The number of batch tags that will be cached in memory
+    cached-batches-size: ${CACHED_BATCHES_SIZE:1000}
```

Comment:
MAX_BATCH_COUNT

Reply:
Renamed it to "MAX_BATCH_LOOP_GUARD" as I think this better describes its purpose. But very open here :-)